taskcluster/ci/generate-screenshots/kind.yml

```diff
@@ -5,46 +5,22 @@
 loader: taskgraph.loader.transform:loader
 
 transforms:
-    - ffios_taskgraph.transforms.screenshots:transforms
+    - ffios_taskgraph.transforms.bitrise:transforms
     - ffios_taskgraph.transforms.secrets:transforms
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - build
 
 job-defaults:
+    bitrise-workflow: jlorenzo_L10nScreenshotsTests
+    build-derived-data-path: {artifact-reference: '<build/public/build/target.zip>'}
     description: Generate localized screenshots by delegating the work to bitrise.io
-    run:
-        using: run-commands
-        use-caches: false
-        secrets:
-            by-level:
-                '3':
-                    - name: project/mobile/firefox-ios/bitrise
-                      key: api_key
-                      path: .bitrise_token
-                default: []
-        dummy-secrets:
-            by-level:
-                '3': []
-                default:
-                      - content: "faketoken"
-                        path: .bitrise_token
+    dependencies:
+        build: build-screenshots
     run-on-tasks-for: []
-    worker:
-        artifacts:
-            - type: file
-              name: public/logs/bitrise.log
-              path: /builds/worker/checkouts/src/bitrise.log
-        docker-image: {in-tree: screenshots}
-        max-run-time: 3600
-    worker-type: b-linux
 
 jobs:
     fr:
         locale: fr
-        worker:
-            # TODO: Move artifacts in transforms once we start supporting all locales
```

Comment:
Note: I actually just did it. I didn't enable all locales but I needed to factorize common arguments for both `fr` and `en-US`.